CitationStyleLanguagePlugin.php

```diff
@@ -426,14 +426,14 @@ public function getCitation(PKPRequest $request, Submission $submission, string
             $citationData->type = ($this->application === 'ojs2' ? 'article-journal' : 'article');
             $citationData->risType = 'JOUR';
             $citationData->id = $submission->getId();
-            $citationData->title = $publication->getLocalizedFullTitle();
+            $citationData->title = htmlspecialchars_decode($publication->getLocalizedFullTitle());
```

Comment:
Now there's a `$format` argument on the `getLocalizedFullTitle()`, but, there's a bug on it, so I think it's better to revert to the previous version and fix the bug on the `getLocalizedFullTitle()` (it's just removing the tags, but leaving the entities...).

Reply:
So you think that function should also do `htmlspecialchars_decode` ?

Reply:
Yes! If I typed `A & B`, the plain text version of it should be `A & B` instead of `A &amp; B`

Reply:
Don't forget that you can specify the type as a second parameter to `PKPPublication::getLocalizedFullText` starting with 3.4.0. You can use `'text'` or `'html'`. However (!!) I think `'text'` mode doesn't actually work as it should -- it should decode entities. @touhidurabir, shouldn't there be a call to `htmlspecialchars_decode` [here](https://github.com/pkp/pkp-lib/blob/main/classes/publication/PKPPublication.php#L104)?

Reply:
Yes, I have just taken a look -- I also think the decoding should happen there.

Reply:
s. https://github.com/pkp/pkp-lib/issues/10135